### ISSUE TITLE
Long Class-Path may break the frontend 

### ISSUE THREAD

**Issue by Flaurite** [jmix-framework/jmix#1571]
### Description
See forum: [discussion](https://forum.jmix.io/t/getting-white-and-blank-screen-with-0-selectlocale-error/2780/3)

Most probably the problem occurs on Windows OS. Vaadin for dev server collects classes with specific annotations to get JS and CSS dependencies.
If classpath is very long, Gradle creates temporary jar file that contains all classpath. However Spring consider this jar without classpath that it contains. So we can see the broken UI.

### Steps to reproduce
1. Create project (1.5+)
2. Add dependencies:
```
implementation 'com.haulmont.yarg:yarg-api:2.2.20'
implementation 'com.haulmont.yarg:yarg:2.2.20'
```
3) Run the application

**AR**
Blank screen and errors in the console.

**Comment by Flaurite** [comment]
As "half" workaround you can try to define application property:
```
vaadin.productionMode=true
```
It allows you to start the application, however some components does not work correctly (at least DatePicker, DateTimePicker, TimePicker, DrawerToggle). Not sure the cause of problem for now. Probably need to create another issue.

**Comment by Flaurite** [comment]
Vaadin suggests the following work arounds:
https://vaadin.com/docs/latest/guide/start/gradle#known-issues

**Comment by Flaurite** [comment]
Fixed only in Release 1.5
